terraform/modules/alertmanager/alertmanager-service.tf

```diff
@@ -175,6 +175,7 @@ data "template_file" "alertmanager_config_file" {
     govuk_pagerduty_key     = data.pass_password.govuk_pagerduty_key.password
     verify_p1_pagerduty_key = data.pass_password.verify_p1_pagerduty_key.password
     verify_p2_pagerduty_key = data.pass_password.verify_p2_pagerduty_key.password
+    dcs_p2_pagerduty_key    = data.pass_password.dcs_p2_pagerduty_key.password
```

Comment:
you'll need to add the `data` resource for this above .... see ~ line 130 to copy what one of the others looks like

Reply:
done